radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNetwork.java

```diff
@@ -51,6 +53,18 @@ public RadixNetwork(PeerDiscovery peerDiscovery) {
 		return this.getStatusUpdates();
 	}
 
+	public Observable<Map<String, RadixClientStatus>> getNetworkState() {
```

Comment:
I suppose this will have to be updated when the `configurable-network` branch has been merged (hopefully today)? Or does this need some other functionality that the `configurable-network` branch's `RadixNetworkState` can't provide?

Reply:
Yes, this is actually more temporary in order to be able to prove some things in the Regression/Acceptance tests.